pkg/network/config/config.go

```diff
@@ -265,6 +265,16 @@ func New() *Config {
 			cfg.Set(join(netNS, "enable_https_monitoring"), true)
 			c.EnableHTTPSMonitoring = true
 		}
+
+		if !cfg.IsSet(join(spNS, "enable_runtime_compiler")) {
+			cfg.Set(join(spNS, "enable_runtime_compiler"), true)
+			c.EnableRuntimeCompiler = true
+		}
+
+		if !cfg.IsSet(join(spNS, "enable_kernel_header_download")) {
+			cfg.Set(join(spNS, "enable_kernel_header_download"), true)
+			c.EnableKernelHeaderDownload = true
+		}
```

Comment:
Do we also want to disable fallback, since we would be requiring runtime compilation?

Reply:
Sounds like that decision is still up in the air.

Reply:
We'll likely defer this to 7.40 and keep the fallback to offset guessing for this release; waiting on the RFC to be finished for the final decision though